main.go

```diff
@@ -142,7 +144,24 @@ func main() {
 			return
 		}
 	}
-	if err := muxServer.ListenAndServe(); err != nil {
+	whitelist, err := env.WhiteList()
+	if err != nil {
+		log.Fatalf("Failed to get whitelist: %v", err)
+	}
+	certManager := autocert.Manager{
+		Prompt: autocert.AcceptTOS,
+		Cache:  autocert.DirCache("certs"),
```

Comment:
Here we should use the `env.CertPath()` method that gets the path from `QUICKFEED_CERT_PATH`.

However, `internal/env/certPath` is currently unexported. But we should exported it.

Reply:
Agreed. I updated the default path to `/internal/config/certs` as the previous default, `/etc/letsencrypt/live` requires elevated privileges to write to.